app/assets/stylesheets/application.css

```diff
@@ -155,3 +155,58 @@ body {
 .all-comments {
   margin-left: 2%;
 }
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍